.jshintignore

```diff
@@ -17,5 +17,6 @@ react/
 # not supersede JSHint but take advantage of advanced language features such as
 # Facebook Flow which are not supported by JSHint.
 modules/translation/translation.js
+app.js
```

Comment:
is this ignored because jshint isn't happy with the es6 code?  i was just running into this elsewhere, i think we should be able to tell jshint that we're using es6.  (or maybe this was for another reason)